Source/Blazorise/Export/ExportResult.cs

```diff
@@ -0,0 +1,20 @@
+namespace Blazorise.Export;
+
+/// <summary>
+/// Represents a generic result of a DataGrid export operation, indicating whether the export was successful.
+/// </summary>
+public interface IExportResult
+{
+    bool IsSuccess { get; init; }
```

Comment:
We usually don't use prefixes like `Is` for public APIs. There might be exception but we tend to not go with it. 

**Little background:** 

`Is` is usually used for internal getters when we need to have some logic behind the public parameter.

Lets say we have 


`[Parameter] public bool Disabled`

then somewhere else, we would have

`protected bool IsDisabled => ParentDropdown?.Disabled ?? Disabled;`

Reply:
This is heavily under my skin and I always forget...